apps/ssr-tests-v9/src/utils/getImportsFromIndexFile.ts

```diff
@@ -0,0 +1,85 @@
+import * as Babel from '@babel/core';
+import * as fs from 'fs';
+import * as path from 'path';
+
+type StoryImport = {
+  local: string;
+  imported: string;
+  path: string;
+};
+
+// TODO: Once will be solved, can be simplified https://github.com/microsoft/fluentui/issues/23393
+function getImportPrefix(filename: string): string {
```

Comment:
This seems really dodgy as an identifier. I would propose to parse the main stories file for the default export and use the literal of the `component` property

```ts
export default {
  title: 'Components/Menu',
  // 👇👇 use this literal
  component: Menu,
  parameters: {
    docs: {
      description: {
        component: [descriptionMd, bestPracticesMd].join('\n'),
      },
    },
  },
};
```

Reply:
- Once #23393 will go in - it will be unified approach and it will be even safer to rely on directory name
- If we will have duplicate identifiers the test will throw, so we have a protection from unexpected issues
- I am not sure that the literal is safe enough as nothing stops someone from doing:
  ```ts
  import { Menu as Component } from './Menu'

  export default {
    component: Component,
  }
  ```

Reply:
makes sense 👌